File: java_rest_Course/Day1/app/src/main/java/com/raindeer/day1/MainActivity.java
package com.raindeer.day1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    public static final String EXTRA_MESSAGE ="com.raindeer.day1.activity_main2";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

   public void sendMassage(View view) {
        Intent intent = new Intent(this, MainActivity2.class);
        EditText editText = (EditText) findViewById(R.id.editTextTextPersonName);
        String message = editText.getText().toString();
        intent.putExtra(EXTRA_MESSAGE, message);
        startActivity(intent);
    }
}

File: java_rest_Course/myappusingKotlin/app/src/main/java/com/codeaxe/myapplication/MainActivity.kt
package com.codeaxe.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Gravity
import android.view.View
import android.widget.Button
import android.widget.ImageButton
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    val buttonClick = findViewById<Button>(R.id.bbuttonimage)
        buttonClick.setOnClickListener{
            val intent = Intent(this, MainActivity2::class.java)
            val toast = Toast.makeText(applicationContext,"Welcome",Toast.LENGTH_LONG)
            toast.setGravity(Gravity.LEFT,200 , 200)
            toast.show()
            startActivity(intent)
        }
    }
}